Bottom: 084b1c91b1e9fcef7d78a561462c116397331e10
Top:    3c23418696570faacc65cbd09d94564fb67715bc
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-17 02:27:20 +0100

Refresh of add-some-annotations-to-help-o

---

diff --git a/src/polygon1.c b/src/polygon1.c
index 4d36bee..1467ddc 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -93,9 +93,8 @@ int vect_inters2 (Vector A, Vector B, Vector C, Vector D, Vector S1,
 
 #define error(code)  longjmp(*(e), code)
 
-#warning TODO: Unlikely
 #define MemGet(ptr, type) \
-if (((ptr) = malloc(sizeof(type))) == NULL) \
+if (UNLIKELY (((ptr) = malloc(sizeof(type))) == NULL)) \
     error(err_no_memory);
 
 #undef DEBUG_LABEL
